site/public/templates/grading/EditGradeable.twig

```diff
@@ -17,7 +17,8 @@
 {% endblock %}
 
 {% block footer_content %}
-    <input type="button" class="btn btn-primary" value="Add New Component" onclick="onAddComponent()" />
+    <input type="button" class="btn btn-primary" value="Add New Component" onclick="onAddComponent(false)" />
+    <input type="button" class="btn btn-danger" value="(DO NOT USE) Add New Peer Component" onclick="onAddComponent(true)"  />
```

Comment:
I think we should wait to merge this until after #4954 is merged and then wrap this with a feature flag:
```
{% if feature_flag_enabled('peer_grading') %}
    <input type="button" class="btn btn-danger" value="Add New Peer Component" onclick="onAddComponent(true)"  />
{% endif %}
```

as I have to imagine somebody is going to try and use this even with the warning otherwise.